### PR TITLE
San bernardino use of force scraper

### PR DESCRIPTION
**Pull request by DouglasKrouth**
Single file, reach out if you'd like to refactor or review over a call. This solution was pretty simple.

### CODE REVIEW

**Comment by thejqs** [comment]
Much obliged, @DouglasKrouth. I should be able to give this a first look tomorrow.

File: USA/CA/san_bernardino_county/san_bernardino_uof_scraper.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            for try_value in range(tries):
+                try:
+                    output_file_name = Path(file_link).name
+                    with open("./data/{}".format(output_file_name), "wb") as f:
+                        response = requests.get(file_link)
+                        # Write to pdf
+                        f.write(response.content)
+                    files_collected_incr += 1
+                    print(
+                        "Retrieved {}. ({} of {} files collected)".format(
```
Comment (thejqs) [inline USA/CA/san_bernardino_county/san_bernardino_uof_scraper.py:50]: One of these numbers seems to be off by one in the scraper output, but which one, ¯\_(ツ)_/¯

An example from the scraper output:
`Retrieved OIS-Arias_Araceli.pdf. (58 of 57 files collected)`

Comment (DouglasKrouth) [reply USA/CA/san_bernardino_county/san_bernardino_uof_scraper.py:50]: Added a +1 to line 50, fixed the issue :)

**Comment by josh-chamberlain** [comment]
Linking to #206

**Review by mbodeantor — APPROVED** [review]
LG!

**Pull request merged**